sprint6/zadanie1.js

```diff
@@ -0,0 +1,20 @@
+/*
+  Zadanie 1. Stwórz funkcję, zadeklaruj w niej tablicę kilku liczb całkowitych,
+  oblicz sumę liczb z tablicy i wypisz do konsoli tę sumę.
+*/
+
+  function sumNumber() {
+    let arr = [1,2,3,4,5,6,7,8,9,10];
+    let suma = 0;
+    for (let i = 0; i < arr.length; i++) {
+        suma = suma + arr[i];
+    }
+    console.log(suma);
+  }
+
+  /*Stworzyłam funkcję o nazwie "sumNumber".
+  Zadeklarowałam dwie zmienne o nazwie arr i sum. Zmienna "arr",
+  w którym tablica przechowuje wartość liczb całkowitych,
```

Comment:
`Zmienna arr, w którym tablica ...` - zmienna `arr` to jest tablica i ewentualnie ona może coś przechowywać.